Assets/MXR.SDK/Plugins/Android/NativeUtils.java

```diff
@@ -194,4 +194,29 @@ public String getInstalledAdminAppPackageName() {
         return null;
     }
 
+    public int getInstalledAdminAppVersionCode() {
```

Comment:
Is there a reason why this function begins with a lower case and the one below, `GetAdminAppVersionCode` begins with an upper case ?

Reply:
One of these is Java, and one of these is C#! These are just different norms based on languages. Comparing file to file (and language to language) these are consistent